cEP-0024.md

```diff
@@ -0,0 +1,100 @@
+# Implement GitMate automations and plugins for coala
+
+| Metadata |                                                       |
+| -------- | ----------------------------------------------------- |
+| cEP      | 24                                                    |
+| Version  | 1.0                                                   |
+| Title    | Implement Gitmate automations and plugins for coala   |
+| Authors  | Vamshi Krishna Bommerla <mailto:bommvams72@gmail.com> |
+| Status   | Proposed                                              |
+| Type     | Process                                               |
+
+## Abstract
+
+This cEP describes the improvements to the coala workflow and the necessary
+automation to realise it, as a part of the
+[GSoC project](https://summerofcode.withgoogle.com/projects/#5344316227059712).
+
+## Introduction
+
+[GitMate](https://gitmate.io/) is an automation tool for developers. While it
+works well, there are a number of missing features which would save time of
+coala community, for example automatically close PRs from master of forks. Many
+but coala needs more plugins which support coala development workflow. This
+project adds plugins to auto-review or auto-reject PRs, assign issues.
+
+## New Features
+
+### Auto-assign issue when it is mentioned in any of PR commits
+
+When a PR is opened in the repo and if any of its commit message
+mentions (Fixes, Closes) any unassigned issues and there is no CI failure, auto
+assign commit author to the mentioned unassigned issues. Comment on the
```

Comment:
This still doesnt describe what happens if the person isnt an org member and thus can not be assigned without some other step being done.

Reply:
If Person isn't member of the org, he/she will not be assigned to any of the issues.